src/Microsoft.VisualStudio.ProjectSystem.Managed/Text/LazyStringSplit.cs

```diff
@@ -69,7 +67,7 @@ internal Enumerator(in LazyStringSplit split)
                 _index = 0;
                 _input = split._input;
                 _delimiter = split._delimiter;
-                Current = null;
+                Current = null!;
```

Comment:
I think this makes sense for enumerators. Technically `Current` can be null, but 99.99% of consumers are `foreach`, in which case you won't see that state of the enumerator. You can still see nulls if the sequence has null elements though (which it won't here).